contracts/p1/StRSR.sol

```diff
@@ -76,15 +76,15 @@ abstract contract StRSRP1 is Initializable, ComponentP1, IStRSR, EIP712Upgradeab
     // === Financial State: Drafts ===
     // Era. If drafts get wiped out due to RSR seizure, increment the era to zero draft values.
     // Only ever directly written by beginDraftEra()
-    uint256 internal draftEra;
+    uint256 internal draftEra; // {draftEra}
```

Comment:
Might as well make this public?

Reply:
wasn't quite ready for your review, you may want to make a full pass again